sdk/ios/Configsum/Configsum/Configsum/Configsum.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Configsum.swift
+//  Configsum
+//
+//  Created by Alexandru Savu on 2017-10-04.
+//  Copyright © 2017 Lifesum. All rights reserved.
+//
+
+public typealias failureBlock = (Error) -> Void
+public typealias successBlock = (Bool) -> Void
+
+public struct Configsum {
+    private let dispatcher: Dispatcher
+
+    public init(environment: Environment, token: String) {
+        let httpClient = HTTPClient(environment: environment)
+        let persistence = Persistence(environment: environment)
+        self.dispatcher = Dispatcher(httpClient: httpClient,
+                                     persistence: persistence)
+    }
+    
+    public func getBool(key: String,
```

Comment:
Could we provide SwiftDoc for these public APIs?

Reply:
👍

Reply:
we could. I should've made a TODO about that ;)

Reply:
Docs would still be awesome (but realize that it's not done yet)

Reply:
in order for this PR to be approved we need docs. nice catch